ingeniamotion/wizard_tests/feedback_test.py

```diff
@@ -331,16 +332,23 @@ def get_current_position(self):
 
     @BaseTest.stoppable
     def current_ramp_up(self):
-        max_current = self.mc.communication.get_register(
-            self.RATED_CURRENT_REGISTER,
-            servo=self.servo, axis=self.axis
-        )
+        dict_currents = {
+            "Rated motor current": self.mc.communication.get_register(
+                self.RATED_CURRENT_REGISTER,
+                servo=self.servo, axis=self.axis
+            ),
+            "Drive nominal current": self.mc.communication.get_register(
+                self.MAXIMUM_CONTINUOUS_CURRENT_DRIVE_PROTECTION,
+                servo=self.servo, axis=self.axis
+            )
+        }
+        max_current = min(dict_currents.values())
+
+        self.logger.debug(f'The maximum current is set by: {min(dict_currents, key=dict_currents.get)}')
         # Increase current progressively
-        self.logger.info(
-            "Increasing current to %s%% rated until"
-            " one electrical cycle is completed",
-            self.PERCENTAGE_CURRENT_USED * 100
-        )
+        self.logger.info(f'Increasing current to %s%% rated until one electrical cycle is completed',
+                         {self.PERCENTAGE_CURRENT_USED * 100})
```

Comment:
It should be `f'Increasing current to {PERCENTAGE_CURRENT_USED * 100}% rated until one electrical cycle is completed'` instead